### ISSUE TITLE
Radio Group Select creates keyboard trap

### ISSUE THREAD

**Issue by slowbot** [bloom-housing/bloom#2187]
**Describe the bug**
When a user selects a radio option, they cannot select another option of escape radio set

**To Reproduce**
1. Go to a radio group and make a selection with the keyboard
2. Attempt to select another option

**Expected behavior**
Should be able to select another option

**Desktop (please complete the following information):**
 - OS:BigSur
 - Browser Chrome
 - Version 95.0.4638.69 (Official Build) (x86_64)

**Comment by jaredcwhite** [comment]
@slowbot I wasn't able to reproduce the issue, at least not in this Storybook example: https://dev-storybook-bloom.netlify.app/?path=/story/forms-fieldgroup--field-group-descriptions

**Comment by jaredcwhite** [comment]
@slowbot Are you still seeing this issue? If so, any more context you can provide so I can try to reproduce it?

**Comment by YazeedLoonat** [comment]
I was able to replicate with a listing that had an already selected `How is the application review order determined?` field you don't get the option to switch using keyboard commands
